python/docs/EXAMPLES.rst

```diff
@@ -72,7 +72,7 @@ After authenticating, open the "Send" tab, fill-out all details, then open the "
 
 .. code::
 
-  $ python3 tools/build_tx.py | trezorctl btc sign-tx -
+  $ python3 python/tools/build_tx.py | trezorctl btc sign-tx -
```

Comment:
the "source distribution" is what you get when you download `trezor...zip` from pypi
so the paths in `python/` subdir are relative to that. please revert here & below.

(the change is appropriate in `tools/snippets`)

Reply:
Done. https://github.com/trezor/trezor-firmware/pull/4092/commits/7407bd7c16a9f83563838da03ae323714d7ba7f9